### ISSUE TITLE
How to add role="button" instead of role="img", it is recommended to use role="button" as the data points are interactive.

### ISSUE THREAD

**Issue by madhukar-deque** [highcharts/highcharts#18209]
`<rect x="34.5" y="251.5" width="26" height="39" fill="rgb(108,217,242)" stroke="#333" stroke-width="1" opacity="1" class="highcharts-point highcharts-color-0 highcharts-drilldown-point" style="cursor: pointer; outline: none;" tabindex="-1" role="img" aria-label="1. %^&amp;*, 4. August 2021."></rect>`

REFERENCE: 
W3C HTML5 Recommendation: https://www.w3.org/TR/html5/sec-forms.html#the-button-element
Deque University: https://dequeuniversity.com/class/custom-widgets2/examples/button

BACKGROUND: 
Every user interface control must have a role to convey what type of control it is for screen reader and other assistive technology users. Native HTML elements - such as <button>, <a>, <input>, <select> - already have a role, so nothing more needs to be done. If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add the relevant role(s) using ARIA as well as expected keyboard interactions.

RULE : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

**Comment by madhukar-deque** [comment]
Hope you all have happy vacation. Can I get update on the same?

**Comment by oysteinmoseng** [comment]
Hi @madhukar-deque, apologies for the late reply here. We have actually done some user testing and have some planned improvements to the accessibility of our drilldown functionality, to be rolled out later this year.

In the meantime, you can override the roles after each render as such: https://jsfiddle.net/oysteinmoseng/k5bhs1on/ 

I am closing this issue for now, but please comment to reopen if you encounter issues.

**Comment by madhukar-deque** [comment]
Property 'drilldown' and 'graphic' does not exist on type 'Point'.

**Comment by oysteinmoseng** [comment]
@madhukar-deque These are internal properties that are not public, so if this is a Typescript error you will have to cast to `any` or similar. Or are you getting actual Javascript errors?

**Comment by ferllings** [comment]
Hello,
Is there any newer way to change the point aria role?
The jsfiddle solution doesn't seems to work anymore.

**Comment by oysteinmoseng** [comment]
Hi @ferllings, the workaround in the linked jsFiddle seems to still work fine for me - could you clarify what issues you encounter with it?

**Comment by ferllings** [comment]
I figured out: I was using line chart, and there is a small difference
